### ISSUE TITLE
make mobile view usable

### ISSUE THREAD

**Issue by elYanuki** [Martin-Hausleitner/nochba-website#1]
The mobile view of the page rn is just a scaled down version of the desktop page and therefore very hard to read and the buttons are incredibly small. From what I can see i believe you are missing the line `<meta name="viewport" content="width=device-width, initial-scale=1" />` in your head in order for the framework to do its thing.

also the page title is `**Nochba**` maybe because its copied from markdown or somthing like that where you can use * to make text **thick**.

And the image size on the landing page is a bit weird, since its just 257 x 512 its visible low res and blury even on laptop screens :)

**Comment by elYanuki** [comment]
ohh and a quite important thing i forgot to mention, it doesnt have https and will therfor be blocked on some devices, rank lower in google searches and seem less trustworthy so id recomment to [encryt](https://letsencrypt.org/de/) the page :)

**Comment by Martin-Hausleitner** [comment]
Thank you for bringing these issues to our attention. Unfortunately, the current code for the website is outdated and is not currently in use. We appreciate your suggestions for solutions, but as we are currently in the process of completing our thesis, the website has a low priority for us at the moment. Our main focus is on the development of the app. The "**" in the page title is from GoDaddy and we are currently redirecting the website directly from Framer using "masked redirection", so we are unable to make changes to the website's code at this time. Once the thesis is complete, we will be revamping the website and will certainly be implementing an SSL certificate as well.
